includes/blocks/class-sensei-continue-course-block.php

```diff
@@ -63,8 +63,13 @@ public function render( array $attributes, string $content ) : string {
 		}
 
 		$target_post_id = Sensei_Utils::get_target_page_post_id_for_continue_url( $course_id, $user_id );
+		$target_url     = get_permalink( absint( $target_post_id ?? $course_id ) );
```

Comment:
Just curious, is there any particular case where `get_target_page_post_id_for_continue_url` returns empty or null? I am seeing that it returns the course_id when it doesn't find any other suitable id https://github.com/Automattic/sensei/blob/aa90079f7e8838d5223b1c044ee20a36a0ee6e21/includes/class-sensei-utils.php#L2998

Reply:
Yep, good catch. This check is not needed. I've blindly copied it from the previous logic. 🙈 Fixed in 6126bd24da3e1f49614c12df49e9137015d5ea5d.